### PR TITLE
Update frontend stack description

### PR DESCRIPTION
**Pull request by JollyBrackets**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,69 +40,21 @@ If you would like to become a part of our team and be added to the repository, p
...
+### Dependencies
+- [Vue.js](https://vuejs.org/) - Frontend Framework
+- [Vueifiy](https://vuetifyjs.com/en/) - UI Components Framework
+- [Vuex](https://vuex.vuejs.org/en/) - State Management
+- [Vue-router](https://router.vuejs.org/en/) - Routing
+- [Vue-i18n](https://kazupon.github.io/vue-i18n/en/) - Internalization
+- [Vue-chartjs](https://github.com/apertureless/vue-chartjs) Charts and Graphs
+
+### DEV Dependencies
+- [Vue-CLI](https://github.com/vuejs/vue-cli/blob/dev/docs/README.md) - Initial build setup ([template](https://github.com/vuetifyjs/webpack))
```
Comment (roma219) [inline README.md]: It is not actually a dev dependency, isn't it?

Comment (piggydoughnut) [reply README.md]: if it is not needed to actually run the project than it is a dev dependency, no?

Comment (roma219) [reply README.md]: It is nor needed to run a project, it's just an util to create initial build

Comment (nicksarafa) [reply README.md]: i wouldn't include `vue-cli` inside the dev dependencies

Comment (piggydoughnut) [reply README.md]: @JollyBrackets please remove the vue-cli from the dev-dependencies list and let's merge this :)

**Pull request merged**